README.md

```diff
@@ -61,6 +99,10 @@ For a local PostgreSQL instance with a username and password one could call:
 ```
 TEST_ONLINE=postgresql://postgres:postgres@localhost:5432/postgres prove -l t/*.t t/*.t.js
 ```
+
+### Documentation
+For further documentation, please see **[docs](https://github.com/openSUSE/qem-dashboard/tree/main/docs)** (WIP)
```

Comment:
What's the "WIP" about? Isn't everything WIP which isn't carved into stone? :)

Reply:
true :+1: I'll remove it